spice/lib/utils.py

```diff
@@ -1178,7 +1178,7 @@ def gen_text2cb(self, kbytes):
         script = self.cfg_vm.helper
         dst_script = self.chk_deps(self.cfg_vm.helper)
         params = "--kbytes2cb"
-        size = int(kbytes) * 1024
+        size = int(kbytes) # deleted * 1024 -> size already in kbytes
         cmd = "%s %s %s" % (dst_script, params, size)
```

Comment:
The purpose of your modification put to the git commit comment.
We need to keep clean code.
Remove comment #...... And move it to git commit.